sdk/template/template/review/template.api.md

```diff
@@ -4,9 +4,9 @@
 
 ```ts
 
-import { OperationOptions } from '@azure/core-http';
-import { PipelineOptions } from '@azure/core-http';
-import { TokenCredential } from '@azure/core-http';
+import { CommonClientOptions } from '@azure/core-client';
+import { OperationOptions } from '@azure/core-client';
```

Comment:
These two can be combined into a single import statement

Reply:
Is this file not being auto-modified when building the project? If it's fine to make manual changes in it ok

Reply:
omg, my bad! I didnt realize this was the autogenerated md file!!! I take back my comment!!!